critiquebrainz/data/model/user_test.py

```diff
@@ -148,21 +148,21 @@ def test_reviews_property(self):
         self.assertEqual(len(user.reviews), 0)
 
     def test_avatar_property(self):
-        user = User(display_name=u'Tester', email=u"example@example.org")
+        user = User(display_name=u'Tester', email=u"example@example.org", id=u"ea1c2bf3-f246-d47c-f6a4-e5f6e7b8d9e0")
```

Comment:
Not sure I understand why you specify an `id` there. It's generated when a user is created: https://github.com/metabrainz/critiquebrainz/blob/master/critiquebrainz/data/model/user.py#L16.